### ISSUE TITLE
Removing post-war houses from appearing before their time in temperate climate (Culprit inside!)

### ISSUE THREAD

**Issue by Seriphyn17** [2TallTyler/itl_houses#43]
![image](https://user-images.githubusercontent.com/131037821/232688881-72083277-0fe4-4c48-8c00-c2267e5ba417.png)

Here is a town created **without** ITL in 1860. There appear to be six housing sprites in total...

- Small cottage
- Medium cottage
- Large cottage 
- Double thatched roof house A
- Double thatched roof house B (w/ pond)
- Townhouse triplets A ("two-up, two-down")
- Townhouse triplets B

![image](https://user-images.githubusercontent.com/131037821/232691005-a8fc8a44-d96e-428a-be8e-82ad00174003.png)

Here is a town created **with** ITL in 1860. There are four housing sprites that better fit in a post-war timeperiod. They are...

- South-facing doubles w/ garages
- South-facing single bungalow w/ garage
- East-facing 2-story doubles
- East-facing 1-story doubles

The good news is that the previously listed historical houses (cottages and townhouses) still spawn, but in far fewer number than the aforementioned homes which seem to be the primary type of house that spawns. There's also a single sub-arctic housing sprite that appears in these towns. Maybe intentional?

![image](https://user-images.githubusercontent.com/131037821/232692225-09103f78-3193-4efd-932a-8515bf286bce.png)

Upon digging and experimenting in the scenario editor, I believe I have found the 'culprit'...

![image](https://user-images.githubusercontent.com/131037821/232694941-96c51c7d-052c-4b3d-9a26-feb6d82845ad.png)

All of the circled houses are 'dynamic', with the red/orange set cycling between those 4 sprites previously listed, and the blue/green set cycling between two townhouse triplets. It seems the red and blue houses were added with fractional cargo requirements to support the early game, but with the perhaps unintentional issue of introducing housing sprites designed to be post-automobile into the 19th century. 

In the vanilla game, the red/orange set are coded to appear in 1931 (the orange set in ITL appears in 1945. suggesting how the red set was added later and thus 'broke' the timeline). The blue/green set (the two-up/two-downs as they're actually called) appear from the year 0 in vanilla game, but 1945 in ITL. I would concur with the vanilla game's interpretation; the two-up/two-downs were built well before the 20th century. They would be most appropriate in the inner city/town area, rather than suburb.

Really want to do a historical run with ITL temperate, so hopefully this can get fixed! :D

**Comment by Seriphyn17** [comment]
### TL;DR

Adding the last two building sprites (red and blue circles) to support fractional cargo requirements "broke" the timeline. So...

- **Remove building sprite circled in red completely (all post-automobile houses are intended to spawn in 1931)**
- **Retain building sprite circled in blue. Possibly change spawn date to 1800**
- **Keep orange building sprite as is, or change to 1931 instead of 1945 (per vanilla)**
- **Change spawn date of building sprite circled in green to earlier than 1945 (or whatever works to keep having a fractional cargo requirement house in play)**

I'm 99% sure doing half/all of this should "fix" this! I know these issues do not appear as egregious when running OpenGFX over DOS original (where the garages are not as obvious), which I believe might have led to this oversight :)

**Comment by 2TallTyler** [comment]
Thanks for the report. I'll have to look into this one, as the temperate houses are the ones I did first as one of my first NewGRFs, and the code is a bit of a mess. I've been meaning to rewrite it to match the newer subtropic and subarctic climates, but haven't found the motivation yet. 😕
